lib/reply.js

```diff
@@ -560,11 +564,12 @@ function handleError (reply, error, cb) {
         statusCode: { value: statusCode }
       }))
   } catch (err) {
+    // error is always FST_ERR_SCH_SERIALIZATION_BUILD because this is called from route/compileSchemasForSerialization
```

Comment:
please check this. I tried to "break" the try block above in any way, but I still get an `FST_ERR_SCH_SERIALIZATION_BUILD` because of https://github.com/fastify/fastify/blob/main/lib/route.js#L314
so I removed the `err.message || ''` below

Reply:
Perrfect.